docs/src/main/asciidoc/sqs.adoc

```diff
@@ -1248,6 +1248,36 @@ If the error handler execution succeeds, i.e. no error is thrown from the error
 IMPORTANT: If the message should not be acknowledged and the `ON_SUCCESS` acknowledgement mode is set, it's important to propagate the error.
 For simply executing an action in case of errors, an `interceptor` should be used instead, checking the presence of the `throwable` argument for detecting a failed execution.
 
+==== Immediate Retry Error Handler
+As mentioned in <<Error Handling>>, by default, messages that cause an error in the listener are only retried after the visibility timeout has expired.
+
+To avoid this delay, you can use the built-in `ImmediateRetryAsyncErrorHandler`. This handler sets the message's visibility timeout to zero, making it immediately available for retry.
```

Comment:
Please add "To avoid this delay, starting in version 3.4, ..."

Also, let's please avoid using "you" to keep a more formal tone.

So something like:
"Starting with version 3.4, Spring Cloud AWS SQS includes the `ImmediateRetryAsyncErrorHandler`, which sets the visibility timeout to zero to enable immediate retry of failed messages.".

Makes sense?